nixos/modules/services/web-apps/pihole-web.md

```diff
@@ -0,0 +1,19 @@
+# Pi-hole Web Dashboard {#module-services-web-apps-pihole-web}
+
+The Pi-hole suite provides a web GUI for controlling and monitoring
+[pihole-FTL](index.html#module-services-networking-pihole-ftl).
+
+## Configuration {#module-services-web-apps-pihole-web-configuration}
+
+Example configuration:
+
+```nix
+{
+  services.pihole-web = {
+    enable = true;
+    port = 80;
```

Comment:
There is no `port` option for `pihole-web`. This example should be:
```
    pihole-web = {
      enable = true;
      ports = [
        "80r"
      ];
    };
```

Reply:
Fixed, thanks